baremetal/metal3data_manager_test.go

```diff
@@ -471,7 +474,7 @@ var _ = Describe("Metal3Data manager", func() {
 				ObjectMeta: testObjectMeta,
 			},
 			expectReady:         true,
-			expectedMetadata:    pointer.StringPtr("String-1: String-1\n"),
+			expectedMetadata:    pointer.StringPtr(fmt.Sprintf("String-1: String-1\nprovideruid: %s\n", provideruid)),
```

Comment:
maybe:
```suggestion
			expectedMetadata:    pointer.StringPtr(fmt.Sprintf("String-1: String-1\n provideruid: %s\n", provideruid)),
```
?

Reply:
I think the comment contains some additional formatting characters. Could you add more info. Thanks.

Reply:
I meant a missing space, so it is structed as:
```
String-1: String-1
provideruid: provideruid
```